packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/tasks/tasks.html

```diff
@@ -0,0 +1,96 @@
+<oui-back-button
+    data-previous-page="{{:: 'pci_databases_connectors_tasks_back' | translate }}"
+    data-on-click="$ctrl.goBack()"
+></oui-back-button>
+
+<div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+        <h2
+            class="mb-0"
+            data-translate="pci_databases_connectors_tasks_title"
+            data-translate-values="{name: $ctrl.connector.name}"
+        ></h2>
+        <span class="oui-paragraph">
+            <span>Type: {{$ctrl.connector.connectorInformation.name}}</span>
+            <span>
+                | Version:
+                {{$ctrl.connector.connectorInformation.version}}</span
+            >
+            <span>
+                | Author: {{$ctrl.connector.connectorInformation.author}}</span
+            >
+            <span ng-if="$ctrl.connector.connectorInformation.documentationUrl">
+                |
+                <a
+                    ng-href="{{$ctrl.connector.connectorInformation.documentationUrl}}"
+                    target="_blank"
+                    rel="noopener"
+                    >docs</a
+                >
+            </span>
+        </span>
+    </div>
+    <oui-button
+        variant="primary"
+        size="l"
+        ng-click="$ctrl.goToEdit($ctrl.connector)"
```

Comment:
**nitpick**: Better to use `on-click` with `oui-button` instead of `ng-click`

Reply:
Changed here: https://github.com/ovh/manager/pull/6582/commits/77aa48b3638d44b1f319b162e896eec643698d07